### ISSUE TITLE
Add schema support for multiple praise receivers

### ISSUE THREAD

**Issue by kristoferlund** [givepraise/praise#745]

**Comment by mohammadranjbarz** [comment]
@kristoferlund 
For this issue, I think we should do these items:
- [ ] Change schema to make receivers array
- [ ] Write migration to change all existence records to convert receiver item to receivers array
- [ ] Change create praise to use receiver array instead of creating multiple time

And some other things that we should consider during implementation, like statistics and web panel info

**Comment by kristoferlund** [comment]
@mohammadranjbarz making this change also very much affects quantification of Praise. When we assign quantifiers currently we create quantifications objects referencing this praise. A praise with 3 receivers that should be quantified by three quantifiers now should have 3x3=9 quantifications objects. This all needs to be handled by UI as well.

**Comment by mohammadranjbarz** [comment]
Thanks for mentioning it, I'm not familiar with quantification part of project

**Comment by Vyvy-vi** [comment]
Has this issue been partially completed?

**Comment by mohammadranjbarz** [comment]
> Has this issue been partially completed?

I haven't done anything for this issue

**Comment by kristoferlund** [comment]
@Vyvy-vi @mohammadranjbarz I don't think we should continue with this issue atm as it would require significant testing in many parts of the system and no immediate benefit to users.
